docs/install_guides/installing-openvino-from-archive-windows.md

```diff
@@ -11,7 +11,8 @@ You can also check the [Release Notes](https://software.intel.com/en-us/articles
 @sphinxdirective
 .. tab:: Operating Systems
 
-  Microsoft Windows 10, 64-bit
+  * Microsoft Windows 10, 64-bit
+  * Microsoft Windows 11, 64-bit
```

Comment:
can we also highlight architecture we are targeting? E.g. explicit x86_64
This is a general recommendation for all the instructions, because OpenVINO also supports ARM